### PR TITLE
UBERF-6626: More detailed info about maintenance

### PR DESCRIPTION
**Pull request by haiodo**
UBERF-6626: More detailed info about maintenance

* Refactor account pod into service for proper use

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NjIxNTNlYTEyMTRhOWI4YTEyZTQ2YjgiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.hnczRhvqohCPi1uPMnSXms3r4dFDdl-ug7h03RqmeE0">Huly&reg;: <b>UBERF-6627</b></a></sub>

### CODE REVIEW

File: packages/core/src/client.ts [hunk 1]
```diff
@@ -82,7 +82,8 @@ export enum ClientConnectEvent {
 
   // Client could cause back a few more states.
   Upgraded, // In case client code receive a full new model and need to be rebuild.
-  Refresh // In case we detect query refresh is required
+  Refresh, // In case we detect query refresh is required
+  Maintenance // In case we detect query refresh is required
```
Comment (BykhovDenis) [inline packages/core/src/client.ts]: Comments

File: server/ws/src/server_http.ts [hunk 2]
```diff
@@ -223,11 +223,12 @@ export function startHttpServer (
       if ('error' in session) {
         void ctx.error('error', { error: session.error?.message, stack: session.error?.stack })
       }
-      await cs.send(ctx, { id: -1, result: 'upgrading' }, false, false)
+      await cs.send(ctx, { id: -2, result: { state: 'upgrading', stats: (session as any).upgradeInfo } }, false, false)
```
Comment (BykhovDenis) [inline server/ws/src/server_http.ts]: Why we need different negative ids?

**Review by BykhovDenis — DISMISSED** [review]

File: packages/core/src/client.ts (hunk 1 above)
Comment (haiodo) [reply packages/core/src/client.ts]: Thanks.

File: server/ws/src/server_http.ts (hunk 2 above)
Comment (haiodo) [reply server/ws/src/server_http.ts]: No need, will revert.

**Review by BykhovDenis — APPROVED** [review]

**Pull request merged**